tests/cycle_utils.py

```diff
@@ -45,10 +46,11 @@ def mock_download_boosts(*args, **kwargs):
 
 
 def mock_upload_boosts(boosts, chain: str):
-    for user in list(boosts["userData"].keys()):
-        # will throw key error if multipliers data not added
-        boosts["userData"][user]["multipliers"]
-    with open("eth-boosts.json", "w") as f:
+    keys = list(boosts["userData"].keys())
+    for user in keys:
+        for _, v in boosts["userData"][user].items():
+           assert type(v) != Decimal
```

Comment:
I think we can make this more complete, we should check to see if this value is json serializable and assert there is no error rather than just checking to make sure it is Decimal

Reply:
the next part does that when it calls json.dump, but we could be more granular